tests/config/singleprovider.e2e.template.config.yaml

```diff
@@ -27,4 +27,5 @@ ols_config:
 dev_config:
   llm_params:
     temperature: 0
-  disable_auth: true
\ No newline at end of file
+  disable_auth: true
+  tls_disabled: true
```

Comment:
just nitpick (or hard read for me): there are two "disable-X" configuration options, each one spelled differently ("disable-a" and "b-disabled")

Reply:
resolved